pytket/extensions/qiskit/backends/aer.py

```diff
@@ -696,7 +696,9 @@ def _qubitpauliop_to_sparsepauliop(
     for term, coeff in operator._dict.items():
         termmap = term.map
         strings.append(
-            "".join(termmap.get(Qubit(i), Pauli.I).name for i in range(n_qubits))
+            "".join(
```

Comment:
Do we want to have some testcase for this?

Reply:
That would be good, though I wasn't sure how best to make a simple test (so I lazily asked Iakov). This method is onlt exposed through `get_operator_expectation_value()`...

Reply:
Iakov may be able to provide us with a test case. Meanwhile I will merge this so that we can do a hotfix.